extensions/amp-analytics/0.1/instrumentation.js

```diff
@@ -557,12 +562,15 @@ export class AnalyticsGroup {
    */
   addTrigger(config, handler) {
     let eventType = dev().assertString(config['on']);
+    let trackerType = eventType;
     // TODO(dvoytenko, #8121): Cleanup visibility-v3 experiment.
-    if (eventType == 'visible' && this.visibilityV3_) {
-      eventType = 'visible-v3';
+    if ((eventType == 'visible' || eventType == 'hidden')
+        && this.visibilityV3_) {
+      eventType += '-v3';
+      trackerType = 'visible-v3';
     }
-    let trackerProfile = EVENT_TRACKERS[eventType];
-    if (!trackerProfile && !isEnumValue(AnalyticsEventType, eventType)) {
+    let trackerProfile = EVENT_TRACKERS[trackerType];
```

Comment:
Let's instead have new `EVENT_TRACKERS['hidden']`, it should be the same as `visible`, specifically it should have `name = 'visible-v3'`. Then the root will reuse the same tracker for both `visibile` and `hidden`. But I'd like to have all trigger event types to be as separate keys of `EVENT_TRACKERS`. On the plus side, you won't need `trackerType` var.